src/afl-analyze.c

```diff
@@ -795,8 +796,10 @@ static void usage(u8 *argv0) {
       "              (must contain abort_on_error=1 and symbolize=0)\n"
       "MSAN_OPTIONS: custom settings for MSAN\n"
       "              (must contain exitcode="STRINGIFY(MSAN_ERROR)" and symbolize=0)\n"
-      "AFL_PRELOAD: LD_PRELOAD / DYLD_INSERT_LIBRARIES settings for target\n"
       "AFL_ANALYZE_HEX: print file offsets in hexadecimal instead of decimal\n"
+      "AFL_MAP_SIZE: the shared memory size for that target. must be >= the size\n"
```

Comment:
`AFL_MAP_SIZE` or `AFL_MAPSIZE`? I've seen both supported in this pr, maybe decide on exactly one?

Reply:
documented is AFL_MAP_SIZE however I code to expect typos and hence AFL_MAPSIZE is secretly also supported (eg. we also support AFL_LLVM_LAF_TRANSFORM_COMPARES and LAF_TRANSFORM_COMPARES, etc)